build-tools/Xamarin.Android.Tools.BootstrapTasks/Xamarin.Android.Tools.BootstrapTasks/StartAndroidEmulator.cs

```diff
@@ -78,8 +78,13 @@ void Run (string emulator)
 			var p = new Process () {
 				StartInfo = psi,
 			};
+			var sdkHome = Path.Combine (AvdManagerHome, "sdk");
```

Comment:
The problem with this is that we're kinda hardcoding the Android SDK directory, and where it is relative to `$(AvdManagerHome)`.  The problem with this is that we allow the Android SDK directory to be overridden via the `$(AndroidSdkDirectory)` MSBuild property, which could be anywhere.

At this point, we should probably add an `AndroidHome` property, so that we can use `<StartAndroidEmulator AndroidHome="$(AndroidSdkDirectory)" … />`.